src/components/Signup/Signup.jsx

```diff
@@ -0,0 +1,255 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+import Joi from 'joi-browser';
+import PropTypes from 'prop-types';
+import signupInputSchema from '../../schemas/signup';
+import signupAction from '../../actions/user/signupAction';
+import Form from '../Form/Form';
```

Comment:
can you create an index file for all our reusable components and import from there

Reply:
This has been handled in Juwon's PR. However, I added it in my PR in the case my PR gets merged before his.